### PR TITLE
'Send & Receive Dogecoin' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Please ensure that the link is to a valid URL.

### CODE REVIEW

File: content/dogepedia/articles/send-and-receive-dogecoin.md [hunk 1]
```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-How can a shibe spend and use his/her Dogecoin? Explain basic functionality and concepts.
\ No newline at end of file
+Sending and receiving Dogecoin is very easy to do. Whether you intend to send or receive Dogecoin, in both cases you will need a [wallet](https://dogecoin.com/dogepedia/articles/how-do-i-get-a-wallet).
```
Comment (MishaBoar) [inline content/dogepedia/articles/send-and-receive-dogecoin.md:8]:
```suggestion
Sending and receiving Dogecoin is very easy to do. Whether you intend to send or receive Dogecoin, in both cases you will need a [wallet](/dogepedia/articles/how-do-i-get-a-wallet).

```

Comment (MishaBoar) [reply content/dogepedia/articles/send-and-receive-dogecoin.md:8]: Relative path

File: content/dogepedia/articles/send-and-receive-dogecoin.md [hunk 2]
```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
...
+
+When sending Dogecoin from your wallet you will need to input a destination wallet address. This is a unique alphanumeric identifier that looks a bit like gibberish! Rest assured, though, a wallet address is certainly *not* gibberish – but specifies a unique destination on the Dogecoin blockchain to which Dogecoin can be sent and stored.  
+
+A lot of wallets also provide a QR code in which the destination wallet’s address is embedded into the QR code. When sending Dogecoin, it is simply a matter of scanning the QR code — or inputting the alphanumeric wallet address — entering the amount of Doge you wish to send, and then clicking ‘send’ or ‘transfer’, dependent on your wallet provider.  
+
+If receiving Dogecoin, you will need to supply the sender with your Dogecoin wallet address and/or the QR code for your wallet.  
+
+One of the (many) great things about Dogecoin is that the transaction speed is significantly quicker than other cryptocurrencies – so you won't have long to wait until the transfer is confirmed. The fees to transfer are also extremely low comparative to other cryptocurrencies, too, so what are you waiting for? Send someone some Doge!
```
Comment (MishaBoar) [inline content/dogepedia/articles/send-and-receive-dogecoin.md:16]:
```suggestion
One of the (many) great things about Dogecoin is that the transaction speed is significantly quicker than other cryptocurrencies – so you won't have long to wait until the transfer is confirmed. The fees to transfer are also extremely low in comparison with other cryptocurrencies. So, what are you waiting for? Send someone some Doge!
```

**Review by MishaBoar — CHANGES_REQUESTED** [review]
All great, just a couple of minor changes I suggested.

**Review by RichieJohn — COMMENTED** [review]
This all looks good to me. Is there anything else I need to do, @tjstebbing or @MishaBoar for this to be merged?

**Review by MishaBoar — APPROVED** [review]

**Pull request merged**